aptos-move/framework/aptos-framework/sources/code.move

```diff
@@ -84,6 +88,18 @@ module aptos_framework::code {
         from.policy <= to.policy
     }
 
+    /// Initialize package metadata for Genesis.
+    fun initialize(aptos_framework: &signer, package_owner: &signer, metadata: PackageMetadata)
+    acquires PackageRegistry {
```

Comment:
Feels like we'll need a formatter someday.  Not there's anything wrong with this format, but I've seen different elsewhere.

I would probably put these on the same line or indent the second line